### PR TITLE
community/libcec: update to 4.0.2

### PR DESCRIPTION
**Pull request by vrnagy**
It depends on p8-platform which is in edge/testing.

### CODE REVIEW

**Comment by vrnagy** [comment]
Build failed because the p8-platform in edge/testing

File: community/libcec/APKBUILD [hunk 1]
```diff
@@ -1,54 +1,37 @@
...
 
 package() {
-	cd "$_builddir"
-        make DESTDIR="$pkgdir" install || return 1
+	cd "$builddir/build"
+	make DESTDIR="$pkgdir" install || return 1
+
+	# Add cec directory to sys.path to get rid of issue https://github.com/Pulse-Eight/libcec/issues/316
+	python_dir=$(python3 -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
+	echo "cec" > "${pkgdir}${python_dir}/cec.pth"
```
Comment (vrnagy) [inline community/libcec/APKBUILD]: Without this line the module does not work. It's a bug in swig/python. Original issue: https://github.com/Pulse-Eight/libcec/issues/316

**Comment by ncopa** [comment]
we can not depend on anything in testing. This means p8-platform (and its dependencies) needs to move to community.

**Comment by vrnagy** [comment]
Fortunately p8-platform does not depends on anything special. Can I move the p8-platform to community in thi PR, or I should open a new PR for it?

**Comment by ncopa** [comment]
do it in this PR

**Comment by vrnagy** [comment]
Moved but the test still failing. Is it expected or I'm missing something?

**Pull request merged**